src/quo2/components/switchers/base_card/style.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo2.components.switchers.base-card.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn base-container
+  [customization-color]
+  {:width            160
+   :height           160
+   :border-radius    16
+   :background-color (colors/custom-color-by-theme customization-color 50 50 40 40)})
```

Comment:
this function is technically not "theme" safe and is using the old approach. 
In the old theming approach there was a global ratom which is being used within the `theme` helper methods (such as this one).
However the new theming approach expects that the `theme` prop is passed as a parameter as it leads to safer usage.
This function does not handle theme via prop, so better to use `color/theme-colors` which does. 
However, you'll notice here that these cards are always in `dark` theme so there is no need for theme.

`(colors/custom-color- customization-color 50 40)` will suffice

 


Since now we are using a context provider